oap-server/server-bootstrap/src/test/java/org/apache/skywalking/oap/server/starter/config/ApplicationConfigLoaderTestCase.java

```diff
@@ -47,4 +51,13 @@ public void testLoadConfig() {
         Properties properties = (Properties) providerConfig.get("properties");
         assertThat(properties.get("jdbcUrl"), is("jdbc:mysql://localhost:3306/swtest"));
     }
+
+    @Test
+    public void testLoadZipkinReceiverConfig() {
```

Comment:
You are putting codes in the config test, why name it related to Zipkin? You are testing the `list type` config injection.